frontend/Panel.js

```diff
@@ -234,7 +239,16 @@ class Panel extends React.Component {
       );
     }
     if (!this.state.isReact) {
-      return <div style={styles.loading}><h2>Looking for React…</h2></div>;
+      return (
+        <div style={styles.loading}>
+          <h2 style={styles.errorMessage}>Unable to find React…</h2>
```

Comment:
Yea, it's a though one. I'll try to look at other solutions as this can potentially get more confusing.